reference/libxml/functions/libxml-set-external-entity-loader.xml

```diff
@@ -18,8 +18,8 @@
   <para>
    デフォルトの外部エンティティローダーを変更します。
    たとえ <constant>LIBXML_NOENT</constant> が個別のXMLの操作に設定されている場合でも、
-   XXE攻撃を避けるために、任意の外部エンティティの展開を抑制するために使います。
-   <function>libxml_disable_entity_loader</function> を呼び出すことが通常は好ましいです。
+   XXE攻撃を避けるために、任意の外部エンティティの展開を抑制するために使うことができます。
```

Comment:
> This can be used to suppress the expansion of arbitrary external entities to avoid XXE attacks, even when LIBXML_NOENT has been set for the respective operation, 
元の文章にはこう書かれており、 can の要素がかけていると感じたため、「できます」を追加してみました